lib/cocoapods-core/specification.rb

```diff
@@ -223,6 +233,18 @@ def subspec?
 
     # @!group Dependencies & Subspecs
 
+    # @return [Bool] If this specification is a libary (app or test) specification.
+    #
+    def library_specification?
+      !app_specification? && !test_specification?
+    end
+
+    # @return [Bool] If this specification is not a libary (app or test) specification.
+    #
+    def non_library_specification?
+      !library_specification?
```

Comment:
is this needed? seems like just negating `library_specification` should be fine even from an API perspective

Reply:
`non_library_specification` is pulled from more so i figure it was just convenience. Don't feel to strongly about this though

Reply:
👍